Assets/Scripts/SS3D/Systems/Interactions/InteractionController.cs

```diff
@@ -34,42 +35,20 @@ public override void OnStartClient()
 
             _radialView = SystemLocator.Get<RadialInteractionView>();
             _camera = SystemLocator.Get<CameraSystem>().PlayerCamera.GetComponent<Camera>();
-        }
-
-        [ServerOrClient]
-        protected override void HandleUpdate(in float deltaTime)
-        {
-            base.HandleUpdate(in deltaTime);
-
-            if (IsServerOnly || !IsOwner || _camera == null || EventSystem.current.IsPointerOverGameObject())
-            {
-                return;
-            }
-
-            if (Input.GetButtonDown("Primary Click"))
-            {
-                ProcessPrimaryClick();
-            }
-
-            else if (Input.GetButtonDown("Secondary Click"))
-            {
-                ProcessSecondaryClick();
-            }
-
-            if (Input.GetButtonDown("Use"))
-            {
-                ProcessUse();
-            }
+            Controls controls = SystemLocator.Get<InputSystem>().Inputs;
+            controls.Other.PrimaryClick.performed += HandlePrimaryClick;
```

Comment:
I'm concerned with the absence of unsuscribing from events overall. Is there a particular reason for that ?